packages/html/ds/src/link/link.test.ts

```diff
@@ -58,8 +58,10 @@ describe('link', () => {
     });
     const linkElement = screen.getByRole('link');
 
-    expect(linkElement.classList.contains('visited:gi-text-purple-700')).toBe(
-      true,
+    console.log(linkElement.classList);
```

Comment:
nitpick: Must be here mistakenly, can we please remove it? 😄

Reply:
ops, sorry